client/components/Players/PlayerManager.vue

```diff
@@ -295,6 +340,125 @@ export default Vue.extend({
             window.addEventListener('keydown', this.handleKeyPress);
           }
 
+          break;
+        case 'playbackManager/INCREASE_QUEUE_INDEX':
+        case 'playbackManager/DECREASE_QUEUE_INDEX':
+        case 'playbackManager/SET_CURRENT_ITEM_INDEX':
+          this.upNextUserHidden = false;
+
```

Comment:
Anything past this line for the subscription was moved to a Vuex plugin, so it shouldn't be there.